src/main/resources/db/migrations/V001__CreateSchemaTables.sql

```diff
@@ -0,0 +1,223 @@
+    location_code VARCHAR(25)
+);
+
+CREATE TABLE public.rooms (
+    id UUID PRIMARY KEY,
+    room VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.court_region(
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    region_id UUID REFERENCES regions(id) NOT NULL,
+    PRIMARY KEY (court_id, region_id)
+);
+
+CREATE TABLE public.courtrooms(
+    id UUID PRIMARY KEY,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    room_id UUID REFERENCES rooms(id) NOT NULL
+);
+
+CREATE TABLE public.cases (
+    id UUID PRIMARY KEY,
+    court_id UUID REFERENCES courts(id) NOT NULL ,
+    reference VARCHAR(25) NOT NULL, 
+    test BOOLEAN DEFAULT FALSE,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.participants (
+    id UUID PRIMARY KEY ,
+    case_id UUID REFERENCES cases(id) NOT NULL,
+    participant_type PARTICIPANT_TYPE NOT NULL,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.bookings (
+    id UUID PRIMARY KEY,
+    case_id UUID REFERENCES cases(id) NOT NULL,
+    scheduled_for TIMESTAMPTZ NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    modified_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.booking_participant (
+    participant_id UUID REFERENCES participants(id) NOT NULL,
+    booking_id UUID REFERENCES bookings(id) NOT NULL,
+    PRIMARY KEY (participant_id, booking_id)
+);
+
+CREATE TABLE public.users (
+    id UUID PRIMARY KEY,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    organisation VARCHAR(250),
+    phone VARCHAR(50),
+    deleted BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMPTZ DEFAULT NOW(),
+    modified_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE TABLE public.roles (
+    id UUID PRIMARY KEY,
+    name VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE public.permissions (
+    id UUID PRIMARY KEY,
+    name VARCHAR(45) NOT NULL
+);
+
+CREATE TABLE role_permission (
+    role_id UUID REFERENCES roles(id),
+    permission_id UUID REFERENCES permissions(id),
+    PRIMARY KEY (role_id, permission_id)
+);
+
+
+CREATE TABLE public.app_access (
+    id UUID PRIMARY KEY,
+    user_id UUID REFERENCES users(id) NOT NULL,
+    court_id UUID REFERENCES courts(id) NOT NULL,
+    role_id UUID REFERENCES roles(id) NOT NULL,
+    last_access TIMESTAMPTZ,
+    active BOOLEAN DEFAULT TRUE,
+    deleted BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMPTZ DEFAULT NOW(),
+    modified_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+
+CREATE TABLE public.capture_sessions (
+    id UUID PRIMARY KEY,
+    booking_id UUID REFERENCES bookings(id) NOT NULL,
+    parent_recording_id UUID REFERENCES recordings(id) NOT NULL,
```

Comment:
You are referencing recordings table before you have created it here- this doesn't work.
```suggestion
    parent_recording_id UUID NOT NULL,
```
```sql
-- Put this at end of file
ALTER TABLE public.capture_sessions ADD FOREIGN KEY ("parent_recording_id") REFERENCES "recordings" ("id");

```

Reply:
Yep, thanks Lucas, I didn't test out the changes as you can see from the mistakes - thanks for spotting :D